source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -117,6 +117,105 @@ For a more data-efficient browser preload, we also recommend that you define the
 />
 ```
 
+## Image Sizes
+
+First be sure you have a good understanding of how the browser selects what image to load depending on the image rendered width and the image widths available to it in the srcset attribute ([this](https://ericportis.com/posts/2014/srcset-sizes/#part-2) link is a good start).
+
+We have added sensible defaults to image sizes on key components in the Home, Category, Search, Product pages. Figuring the correct `sizes` to set can be a daunting task, you need to know the different sizes available for your image and take into account the size it will be rendered as in relation to media breakpoints on the page and parent containers' `max-width`, `padding`and`margin` properties. To simplify this process we devised a smart way to determine these numbers in a very straightforward way.
```

Comment:
Nit: I think it is better to stay vague here so that we don't have to update it over time ;)
```suggestion
We have added sensible defaults to image sizes on key components in the principal pages. Figuring the correct `sizes` to set can be a daunting task, you need to know the different sizes available for your image and take into account the size it will be rendered as in relation to media breakpoints on the page and parent containers' `max-width`, `padding`and`margin` properties. To simplify this process we devised a smart way to determine these numbers in a very straightforward way.
```